packages/application-config/src/formatters.ts

```diff
@@ -117,7 +118,58 @@ function entryPointUriPathToPermissionKeys<PermissionName extends string>(
   );
 }
 
+type TResourceAccessPermissionName<PermissionName extends string = ''> =
+  | `view${Capitalize<PermissionName>}`
+  | `manage${Capitalize<PermissionName>}`;
+
+const getPermissions = (
```

Comment:
Nit: since we're only mapping some data, I would keep this function in the `transformers.ts` file.

Reply:
Updated [fbd2537](https://github.com/commercetools/merchant-center-application-kit/pull/2822/commits/fbd25376f21bfc7d81da512270a7e3dded6ea308)